third_party/java/arcs/build_defs/native.oss.bzl

```diff
@@ -27,3 +27,9 @@ proto_library = _proto_library
 java_proto_library = _java_proto_library
 
 android_proto_library = _android_proto_library
+
+# In OSS we don't have a need for python library for our protos,
+# but in internal Google repo we do. Instead of adding real python dependencies
+# we use this blank rule. In the internal repo we use real definition.
+def py_proto_library(name, api_version, deps):
```

Comment:
Wouldn't it be better to keep this in sync with internal repo?

Reply:
Never mind. It looks like we don't sync this with internal repo?

Reply:
I don't understand what you're asking exactly.  This file is technically copybared over, but not used internally.